aten/src/ATen/Context.cpp

```diff
@@ -171,6 +171,7 @@ bool Context::userEnabledOverrideableSDP() const {
 }
 
 static constexpr const auto cublas_config_var_name = "CUBLAS_WORKSPACE_CONFIG";
+static constexpr const auto hipblaslt_allow_tf32 = "HIPBLASLT_ALLOW_TF32";
```

Comment:
Is there a distinction between HIPBLASLT/HIPBLAS? For CUDA backends the switch would apply to both

Reply:
Yeah, but I don't think we care too much about hipblas because it is very very slow on MI300 and afaiu all the new development are on hipblaslt